wpa_supplicant/ctrl_iface_zephyr.c

```diff
@@ -12,25 +12,21 @@ static void wpa_supplicant_ctrl_iface_receive(int sock, void *eloop_ctx,
 					      void *sock_ctx)
 {
 	struct wpa_supplicant *wpa_s = eloop_ctx;
-	char *buf, *pos;
+	char buf[CTRL_IFACE_MAX_LEN], *pos;
```

Comment:
(1) Should we Initalise here "{0}" instead of doing memset. 
(2) We are allocating 1 byte less, Will it impact anything?

Reply:
1 - No, for a on-stack buffer both are same (compiler internally does `memset`: see https://stackoverflow.com/a/453462)
2 - done, thanks.